docs/user-guide/src/capm3/pivoting.md

```diff
@@ -0,0 +1,170 @@
+* **Maintain connectivity towards provisioning network.**
+
+  Baremetal machines boot over a network with a DHCP server. This requires
+  maintaining a fixed IP end points towards the provisioning network. This is
+  achieved through _keepalived_. A new container is added namely
+  _ironic-endpoint-keepalived_ in the ironic deployment which maintains the Ironic Endpoint using
+  keepalived. The motivation behind maintaining Ironic Endpoint with Keepalived
+  is to ensure that the Ironic Endpoint IP is also passed onto the target
+  cluster control plane. This also guarantees that once moving is done and the
+  management cluster is taken down, target cluster controlplane can re-claim
+  the Ironic endpoint IP through keepalived. The end goal is to make Ironic
+  endpoint reachable in the target cluster.
+
+* **BMO is deployed as part of CAPM3.**
+
+  If not, it has to be deployed before the `clusterctl init` and the BMH CRDs
+  need to be labeled accordingly manually. Separate labeling for BMH CRDs is required
+  because since CAPM3 version [v0.5.0](https://github.com/metal3-io/cluster-api-provider-metal3/releases/tag/v0.5.0)
+  BMO/BMH CRDs are not deplopyed as part of CAPM3 deployment anymore.
+  This is a prerequisite for both the management and the target cluster.
+
+* **Objects should have a proper owner reference chain.**
+
+  `clusterctl move` moves all the objects to the target cluster following the
+[owner reference chain](https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#ownerreferences-chain).
+So, it is necessary to verify that all the desired objects that needs to
+be moved to the target cluster have a proper owner reference chain.
+
+## Step by step pivoting process
+
+As described in
+[clusterctl](https://cluster-api.sigs.k8s.io/clusterctl/commands/move.html)
+the whole process of bootstrapping a management cluster to moving
+objects to target cluster can be described as follows:
+
+The move process can be bounded with the creation of a temporary bootstrap
+cluster used to provision a target management cluster.
+
+This can now be achieved with the following procedure:
+
+1. Create a temporary bootstrap cluster, the temporary bootstrap cluster (source cluster)
+ could be created tools like e.g. using Kind or Minikube using and after the source cluster
+ is up and running then the CAPI and provider components can be installed with `clusterctl` to
+ the source cluster.
+
+2. Install Ironic  components namely: ironic, ironic-inspector,
+   ironic-endpoint-keepalived, httpd and dnsmasq.
+
+3. Use clusterctl init to install the provider components
+
+   Example:
+
+   `clusterctl init --infrastructure metal3:v1.1.0
+   --target-namespace metal3 --watching-namespace metal3`
+
+ This command will create the necessary CAPI controllers (CAPI, CABPK, CAKCP)
+ and CAPM3 as the infrastructure provider. All of the controllers will be installed
+ on namespace `metal3` and they will be watching over objects in namespace `metal3`.
```

Comment:
There is indentation issue if check https://deploy-preview-254--metal3io.netlify.app/capm3/pivoting.html
```suggestion
     This command will create the necessary CAPI controllers (CAPI, CABPK, CAKCP)
     and CAPM3 as the infrastructure provider. All of the controllers will be installed
     on namespace `metal3` and they will be watching over objects in namespace `metal3`.
```